tests/AdaptiveExpressions.Tests/RewriteTests.cs

```diff
@@ -1,33 +1,12 @@
 ﻿using System.Collections.Generic;
-using AdaptiveExpressions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
 
 namespace AdaptiveExpressions.Tests
 {
-    [TestClass]
     public class RewriteTests
     {
         public static IEnumerable<object[]> Data => new[]
         {
-            /*
-            Test("woof.blah", "woof.blah"),
-            Test("!woof.blah", "!woof.blah"),
-            Test("!!woof.blah", "woof.blah"),
-
-            // Comparisons
-            Test("!(woof.blah < 3)", "woof.blah >= 3"),
-            Test("!(woof.blah <= 3)", "woof.blah > 3"),
-            Test("!(woof.blah == 3)", "woof.blah != 3"),
-            Test("!(woof.blah != 3)", "woof.blah == 3"),
-            Test("!(woof.blah >= 3)", "woof.blah < 3"),
-            Test("!(woof.blah > 3)", "woof.blah <= 3"),
-
-            // Logical
-            Test("bark == 1 && bark == 2", "bark == 1 && bark == 2"),
-            Test("bark == 1 || bark == 2", "bark == 1 || bark == 2"),
-            Test("(bark == 1 || bark == 2) && (arg == 3 || arg == 4)", 
-                "or(bark == 1 && arg == 3, bark == 1 && arg == 4, bark == 2 && arg == 3, bark == 2 && arg == 4)"),
-            */
```

Comment:
These tests look reasonable.  Are they found somewhere else?

Reply:
These tests were commented out so I removed them. Since we don't check tests in I don't know who or why they were commented out.